src/test/java/com/github/timtebeek/KafkaDeadLetterPublishingApplicationTests.java

```diff
@@ -53,7 +54,7 @@ static void setProperties(DynamicPropertyRegistry registry) {
 	@BeforeAll
 	static void setup() {
 		// Create a test consumer that handles <String, String> records, listening to orders.DLT
-		// https://docs.spring.io/spring-kafka/docs/2.8.1/reference/html/#testing
+		// https://docs.spring.io/spring-kafka/docs/current/reference/html/#testing
```

Comment:
Might seem odd at first, but I prefer to link to exact version numbers at stable URLs, rather than the moving target "current", which will mean something entirely different a year from now. It's happened before that links would break when using `current`.
```suggestion
		// https://docs.spring.io/spring-kafka/docs/3.0.x/reference/html/#testing
```